client/src/pricing_mod1/ModOne.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import PricePerNight from './PricePerNight';
+import Rating from './Rating';
+
+export const Holder = styled.div`
+  display: flex;
+  justify-content: center; 
+  align-items: center;
+  flex-direction: column;
+  width: inherit;
+  width: 250px;
+  border-bottom: 1px solid rgb(172, 172, 172);
+  font-weight: bold;
+`;
+Holder.displayName = 'Holder';
+
+export default function ModOne( props ) {
+  ModOne.propTypes = {
+    price: PropTypes.number,
+    rating: PropTypes.number,
+    numReviews: PropTypes.number,
+
+  };
+  ModOne.defaultProps = {
+    price: 0,
+    rating: 0,
+    numReviews: 0,
+  };
+  if ( props.price ) {
+    return (
+      <div>
+        <Holder>
+          <PricePerNight price={props.price} />
+          <Rating rating={props.rating} numReviews={props.numReviews} />
+        </Holder>
+      </div>
+    );
+  }
+  return null;
```

Comment:
You used defaultProps in case no props were passed down.  But because 0 is a falsy value, you'll return 'null' if no props are passed down anyways.  So either using defaultProps is redundant if you want to return null in the case of no props passed down, or you should choose to return _something_ (not null) if price = 0.

Reply:
The linter yells at you if you don't, so you have to. Either way, I need it to return null if no props are passed down.

Reply:
Got it!  Makes sense.